### PR TITLE
added global nuxt type

### PR DESCRIPTION
**Pull request by victorgarciaesgi**
Adding NuxtApp type to global const `$nuxt`

### CODE REVIEW

File: packages/types/app/index.d.ts [hunk 1]
```diff
@@ -100,3 +100,8 @@ export interface NuxtApp extends Vue {
   refresh(): void
   setLayout(layout: string): any // TBD
 }
+
+// Global Nuxt Object accessible outside components
+declare global {
+  const $nuxt: NuxtApp
```
Comment (kevinmarrec) [inline packages/types/app/index.d.ts]: I think that it should not be declared as variable but more likely as the correct property of `window` (`window.$nuxt`), so directly around the global `Window` interface from DOM types.

```suggestion
  interface Window {
    $nuxt: NuxtApp
  }
```

Comment (victorgarciaesgi) [reply packages/types/app/index.d.ts]: @kevinmarrec Have you tested it? I have build error with your solution :(

Comment (kevinmarrec) [reply packages/types/app/index.d.ts]: Yes. You should use `window.$nuxt` instead of just `$nuxt` in your application.

Comment (kevinmarrec) [reply packages/types/app/index.d.ts]: image

**Pull request closed without merging**